monix-catnap/src/main/scala/com/avast/sst/monix/catnap/CircuitBreakerConfig.scala

```diff
@@ -0,0 +1,8 @@
+package com.avast.sst.monix.catnap
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class CircuitBreakerConfig(maxFailures: Int,
+                                      resetTimeout: FiniteDuration,
+                                      exponentialBackoffFactor: Double = 1.0,
```

Comment:
I have 0 experience with Monix's circuit breaker, but just from the name, `exponentialBackoffFactor`, isn't value `1.0` completely non-sensical?

Reply:
This is the default. I guess the author of Monix decided that 1 is a good starting value.

Reply:
fair enough